src/routes.js

```diff
@@ -156,6 +156,7 @@ export default (
 
       <BMProtectedRoute path="/bmdashboard" exact component={BMDashboard} />
       <Route path="/bmdashboard/login" component={BMLogin} />
+      <BMProtectedRoute path="/bmdashboard/materials-list" component={MaterialsList} />
```

Comment:
Hi Tim, while checking Figma, in order for the path in the navbar header component to match with this, it should be "/bmdashboard/material-list," right?

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121908988/d66bf120-cc52-4ed0-a925-9f0409c9a6be)